### PR TITLE
chore: consolidate deploy scripts

### PR DESCRIPTION
**Pull request by ryanchristo**
Closes: #39

This pull request consolidates the deploy scripts with the option of setting the environment to `live` or `local`.

### CODE REVIEW

File: scripts/deploy.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+const {
+	DEPLOY_ENVIRONMENT, 				// "live" or "local"
+	OWNER_MNEMONIC, 						// (optional) owner mnemonic
+	BRIDGE_CONTROLLER_ADDRESS,  // bridge controller address (i.e. token issuer)
+	NCT_POOL_ADDRESS,						// NCT pool address
+} = process.env;
+
+async function deploy() {
+	assert.ok(DEPLOY_ENVIRONMENT, "DEPLOY_ENVIRONMENT environment variable must be set");
```
Comment (pheuberger) [inline scripts/deploy.js:17]: i'm curious, why are you using `assert.ok` in favor of plain `assert`?

File: scripts/deploy.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+}
+
+async function deployLocal() {
+	const [adminAccount, bridgeAccount] = await hre.ethers.getSigners();
+
+	console.log(`Deploying fixed toucan contracts...`);
+
+	// registry, projects, vintages, batches, tco2Factory, nctPool
+	const env = await deployFixedContracts(adminAccount);
+	let contracts = {};
```
Comment (pheuberger) [inline scripts/deploy.js:53]: nit: can be a `const`

File: scripts/deploy.js [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+		)
+	);
+}
+
+async function getOwner() {
+	if (OWNER_MNEMONIC) return OWNER_MNEMONIC;
+
+	console.log("OWNER_MNEMONIC environment variable not found. Looking for mnemonic.txt...");
+
+	let mnemonic
```
Comment (pheuberger) [inline scripts/deploy.js]: any reason to not return directly from inside the catch block and throw an exception/assert.fail from the catch block?

**Comment by pheuberger** [comment]
generally would prefer commits to already be properly formatted so format commits are not necessary :) also, i was wondering why you merged master into this branch instead of rebasing the branch onto master @ryanchristo

**Review by 0xmichalis — APPROVED** [review]
LGTM, will try to address any feedback from Philipp and other fixes to `.env.example` that I'd like to have in place in a follow-up.

**Pull request merged**